### PR TITLE
Adding instructions for PMC finalization

### PR DESCRIPTION
**Pull request by chamikaramj**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: contributor-docs/release-guide.md [hunk 1]
```diff
@@ -1228,6 +1228,24 @@ Ping [dev@](mailto:dev@beam.apache.org) mailing list for assistance if you need
...
+
+# Copy files from the `dev` artifact repo to the `release` artifact repo.
+
+cd release
+
+svn add $RELEASE_VERSION
+
+svn rm $OLD_RELEASE_VERSION   # Delete all artifacts from old releases.
+
+svn commit -m "Adding artifacts for the ${RELEASE_VERSION} release and removing old artifats"
```
Comment (damccorm) [inline contributor-docs/release-guide.md:1246]:
```suggestion
svn commit -m "Adding artifacts for the ${RELEASE_VERSION} release and removing old artifacts"
```

**Review by damccorm — APPROVED** [review]
Thanks!

File: contributor-docs/release-guide.md [hunk 2]
```diff
@@ -1228,6 +1228,24 @@ Ping [dev@](mailto:dev@beam.apache.org) mailing list for assistance if you need
...
+
+mkdir release/$RELEASE_VERSION
+
+# Copy files from the `dev` artifact repo to the `release` artifact repo.
+
+cd release
+
+svn add $RELEASE_VERSION
+
+svn rm $OLD_RELEASE_VERSION   # Delete all artifacts from old releases.
```
Comment (damccorm) [inline contributor-docs/release-guide.md:1244]: Oh, it looks like maybe this didn't work this time as well? https://dist.apache.org/repos/dist/dev/beam/ still has a version for 2.52.0 (and a couple other versions)

Comment (chamikaramj) [reply contributor-docs/release-guide.md:1244]: This is to remove old artifacts from the release repo. Removing from the dev repo can be done by the release manager when needed (or from any committer).

**Comment by chamikaramj** [comment]
Thanks.

**Pull request merged**